app/javascript/mastodon/components/blurhash.tsx

```diff
@@ -0,0 +1,45 @@
+import { decode } from 'blurhash';
+import React, { useRef, useEffect } from 'react';
+
+type Props = {
+  hash: string;
+  width?: number;
+  height?: number;
+  dummy?: boolean;
+  children?: never;
```

Comment:
I think it would be good to copy comments from .jsx, since `dummy` properties without any document sounds mysterious

Reply:
added!